_includes/guides-header.html

```diff
@@ -27,7 +27,7 @@ <h3 class="logo">
 					</ul>
 
 					<p><a href="https://addons.mozilla.org/developers/addons" class="button" target="_blank" rel="noreferrer noopener">Edit Product Page</a></p>
-					<p><a href="{{ "/" | relative_url }}" class="back">Extension Workshop</a></p>
+					<p><a href="{{ "/" | relative_url }}" class="back">Firefox Extension Workshop</a></p>
```

Comment:
After previewing this on a local build, I think we can keep the header "Extension Workshop" and let the logo communicate the branding. :) 

Can you revert this?